src/pages/chatRoom/chatRoom.svelte

```diff
@@ -100,7 +100,7 @@
         comments = [
 			...comments.slice(0, index),
             newComment,
-			...comments.slice(index + 1, comments.length-1)
+			...comments.slice(index + 1, comments.length)
```

Comment:
Because of this, whenever a comment was being posted when moderation kicks in, it was disappearing.
It was to fix the 3rd requirement as stated by Emma -
"Important point: now when a pre-programmed message has been sent, and we set the moderation intervention to appear 10 seconds after, any message sent by any user during these 10 seconds disappears the moment the moderation appears."